wweval/R/eval_fit.R

```diff
@@ -23,10 +23,10 @@ eval_fit_ww <- function(config_index,
   }
 
 
-  cfaforecastrenewalww::create_dir(output_dir)
-  cfaforecastrenewalww::create_dir(raw_output_dir)
+  wwinference::create_dir(output_dir)
+  wwinference::create_dir(raw_output_dir)
 
-  params <- cfaforecastrenewalww::get_params(params_path)
+  params <- wwinference::get_params(params_path)
```

Comment:
Ok so  I think the issue is that in the `get_stan_data_list()` function (which we will eventually replace with the one in `wwinference` I do this assigning of the column headers to the parameters in the dataframe, instead of calling them explicitly with $ (bad practice I know). 

So this is failing in the run. My suggestion is not to fix it in `wweval`, but perhaps convert `params` to a dataframe temporarily, since the downstream functionality will eventually be replaced.

Reply:
Just addressed that by adding `|> as.data.frame()` right after all calls to `get_params`. The masking warning will go away once this PR is merged, as the functions mentioned in it are no longer in the `cfaforecast...` package. The warnings show up b/c the package is being installed from main, not from this branch.